app/src/main/java/com/github/orkest/domain/DeezerApi.kt

```diff
@@ -0,0 +1,28 @@
+package com.github.orkest.domain
+
+import java.net.URL
+
+interface DeezerApi : HttpHandler{
+
+    fun searchAsong(songName : String, artistName:String) : String{
+        val url = URL("https://api.deezer.com/search/track?q=$songName $artistName")
+         return PostOrGEtAndGetResponse(url,false)
+    }
+
+    fun fetchTheUserId(access_token:String): String{
+        return PostOrGEtAndGetResponse(URL("https://api.deezer.com/user/me?access_token=$access_token"),false)
+    }
+
+    fun createANewPlaylistOnTheUserProfile(userId:String,access_token: String,playlistTitle: String="Orkest") :String{
+        return PostOrGEtAndGetResponse(URL("https://api.deezer.com/user/$userId/playlists?access_token=$access_token&title=$playlistTitle"),true)
+    }
+
+    fun addANewSongToOrkestPlayList(access_token: String,playlistId: String="Orkest",trackId:String):String{
+        return PostOrGEtAndGetResponse(URL("https://api.deezer.com/playlist/$playlistId/tracks?access_token=$access_token&songs=$trackId"),true)
+    }
+
+}
+
+class deezerApiImplemented : DeezerApi{
+
+}
```

Comment:
A few remarks : 

1) Why do you implement the interface HttpHandler but don't declare any methods from it
2) I think it is better practice to use Uri.Builder instead of concatening the string
3) There's a lack of code comments
4) There is no error handling for your network operations

Reply:
1. Because it can be use with the type HTTP handler
2. True
3. added in a following commit
4. The network operations fails will be handle in a higher level